
From: Harm Verhagen <hverhagen@dse.nl>

The kaweth usb ethernet driver is pretty noisy.  It generates over 40 lines
during a modprobe.  The following patch reduces the prints generated by
this driver.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/usb/net/kaweth.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/usb/net/kaweth.c~shut-up-kaweth-usb-net-driver drivers/usb/net/kaweth.c
--- 25/drivers/usb/net/kaweth.c~shut-up-kaweth-usb-net-driver	Wed Jun 23 16:23:07 2004
+++ 25-akpm/drivers/usb/net/kaweth.c	Wed Jun 23 16:23:07 2004
@@ -62,7 +62,7 @@
 #include <asm/semaphore.h>
 #include <asm/byteorder.h>
 
-#define DEBUG
+#undef DEBUG
 
 #ifdef DEBUG
 #define kaweth_dbg(format, arg...) printk(KERN_DEBUG __FILE__ ": " format "\n" ,##arg)
_
